Strings.md

```diff
@@ -0,0 +1,60 @@
+Table of Contents
+=================
+
+   * [Table of Contents](#table-of-contents)
+   * [Strings](#strings)
+      * [Memory Allocation](#memory-allocation)
+      * [Mutable and Immutable](#mutable-and-immutable)
+      * [Basic Operations](#basic-operations)
+
+# Strings
+
+Strings is a data type and it has operations like traverse, copy, get and so on. So these Strings operations involves Time and Space complexity. 
+
+Strings uses Array data structure to store the data inside the memory bound.
+
+## Memory Allocation
+
+All strings are generally represented in English alphabets. So ASCII character encoding standard is used for storing the data in an Array inside memory bound.
+
+```
+ASCII uses 1 byte of 8 bits
+```
+
+In case of different languages say Chinese then it uses different character encoding standards to store the data in an Array which will occupy different memory slots.
+
+## Mutable and Immutable
+
+Strings implementation varies in different languages. In C++ Strings are mutable whereas in other languges like Java, JavaScript, Python ...etc Strins are immutable in nature. Strings behave much like normal arrays, with the main distinction being that, in most programming languages (C++ is a notable exception), strings are
```

Comment:
@shangan23  Change the typo error from Strins to Strings

Reply:
Thanks for identifying. Resolved.